sentry/src/test/java/io/sentry/HubTest.kt

```diff
@@ -1005,6 +1005,17 @@ class HubTest {
         }
     }
 
+    @Test
+    fun `when startTransaction with parent sampled and no traces sampler provided, transaction inherits sampling decision`() {
+        val hub = generateHub()
+        val transactionContext = TransactionContext("name")
+        transactionContext.parentSampled = true
+        val transaction = hub.startTransaction(transactionContext)
+        assertNotNull(transaction)
+        assertNotNull(transaction.isSampled)
+        assertTrue(transaction.isSampled!!)
```

Comment:
do we need `!!` here? Kotlin should do it automatically for you as you did `assertNotNull(transaction.isSampled)` before

Reply:
we wouldn't if its immutable. But unfortunately it is mutable.